File: E2F-Server2/E2F-Server2/Model/Sheet.cs
namespace E2F_Server2.Model
{
    public class Sheet
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public int HeaderStartRow { get; set; }
        public int HeaderEndRow { get; set; }
        public string HeaderStartCol { get; set; } = null!;
        public string HeaderEndCol { get; set; } = null!;
        public int SheetIndex { get; set; }
        public List<SheetColumn> Columns { get; set; } = new();
    }
}

File: E2F-UI2/src/types/Sheet.ts
import { SheetColumn } from "./SheetColumn";

export type Sheet = {
  id: number;
  name: string;
  headerStartRow: number;
  headerEndRow: number;
  headerStartCol: string;
  headerEndCol: string;
  sheetIndex: number;
  columns: SheetColumn[];
};
